.ruby-version

```diff
@@ -1 +1 @@
-2.0.0-p247
+2.0.0-p645
```

Comment:
Have discussed with Paul and Steve and this should be upgraded to 2.2.x once the deployment issues are solved.